cmd/eval.go

```diff
@@ -179,43 +180,60 @@ func determineInputSpec(arguments args) (*appstudioshared.ApplicationSnapshotSpe
 	var appSnapshot appstudioshared.ApplicationSnapshotSpec
 
 	// read ApplicationSnapshot provided as a file
-	if arguments.filepath != "" {
+	filepath := arguments.filepath
+	if filepath != "" {
+		utils.Trace("Reading application snapshot from %s", filepath)
```

Comment:
Consider using `%q` instead of `%s` when the value is a string. This will quote the value making it easier to read. 
```
Reading application snapshot from yolo
```
vs
```
Reading application snapshot from "yolo"
```
I find it's helpful to diagnose issues caused by extra white spaces.